tools/src/command/public_params.rs

```diff
@@ -42,6 +42,7 @@ pub(crate) fn setup_params(args: SetupArgs) -> anyhow::Result<PathBuf> {
     } else {
         match vm_config.prover {
             ProverImpl::Jolt => anyhow::bail!("Jolt doesn't require Nova-setup"),
+            ProverImpl::HyperNova => todo!(),
```

Comment:
Consider returning human readable error like "HyperNova isn't supported by the cli" instead of panicking.

Alternatively, remove it from clap traits so it cannot be possibly parsed.

Reply:
Done.